### PR TITLE
fix: refactor DashBoard page

### PR DESCRIPTION
**Pull request by nguyendang127**

### CODE REVIEW

File: client/src/pages/Dashboard/Dashboard.jsx [hunk 1]
```diff
@@ -23,26 +31,70 @@ const Dashboard = () => {
...
+                  }
+                  key="1"
+                >
+                  <TargetPercent />
+                </TabPane>
+                <TabPane
+                  tab={
+                    <span>
+                      <PieChartOutlined style={{ fontSize: "1rem" }} />
+                      Job's Status
```
Comment (pum-k) [inline client/src/pages/Dashboard/Dashboard.jsx:71]: Job's Status = Task's Status

**Review by pum-k — CHANGES_REQUESTED** [review]

**Comment by pum-k** [comment]
image

Please fix this animation when i change tab

**Comment by nguyendang127** [comment]
Roger that !!!

**Comment by nguyendang127** [comment]
I have sticked the Animate of Motion for you to check , sorry for this problems :(((

**Review by pum-k — APPROVED** [review]

**Pull request merged**